src/ImageSharp/Formats/Tiff/ITiffEncoderOptions.cs

```diff
@@ -36,5 +36,10 @@ internal interface ITiffEncoderOptions
         /// Gets the quantizer for creating a color palette image.
         /// </summary>
         IQuantizer Quantizer { get; }
+
+        /// <summary>
+        /// Gets a value indicating whether preserve metadata.
+        /// </summary>
+        bool PreserveMetadata { get; }
```

Comment:
We normally just check if the `Metadata` property is null.

Reply:
Done!